### DISCUSSION TITLE
How exactly does Site Action Summary work?

### DISCUSSION THREAD

**Discussion by morphinestyle** [akeeba/panopticon#560] [Q&A (Community Support)]
I made an update to an extension of a site, I thought I would find the update reported in the Site Action Summary that I received by email, but it is completely empty.

Am I doing something wrong?

**Comment by nikosdion** [comment]
If you just did the update it won't show up yet. The reports being emailed include actions taken in the past. If you are emailing daily reports it shows actions taken in the previous calendar day of the report. A report emailed at 6 a.m. on Monday would contain the actions taken in Sunday, midnight to 23:59:59.

**Reply by morphinestyle** [reply]
By choosing "monthly" even if I put a daily CRON it should show me the updates made in the last month, correct?

**Reply by nikosdion** [reply]
Correct.

**Reply by morphinestyle** [reply]
I set this setting with a website, therefore: daily CRON with MONTHLY report, I should receive a daily email with the two updates to the extensions made last week (Joomla language, and Akeeba Backup Pro).

I receive the email every day but it is always empty: the email template is all there, but the list of actions performed on the website is empty.

**Reply by nikosdion** [reply]
I am a bit confused about your setup. If it a monthly report it will only show actions taken in the previous calendar month. If you run it anytime between February 1st 00:00:00 and February 29th 23:59:59 it will send an email with actions taken between January 1st 00:00:00 and January 31st 23:59:59.

When you set up a scheduled report it installs a scheduled task which runs once per time period. For a monthly report that should be once a month.

Do I understand correctly that you have _manually_ created a CRON job to run daily for the monthly report? This would be unadvisable, as it will generate the same report every single day of the month. That's not what you really want to happen.

**Reply by morphinestyle** [reply]
Exactly, it's a setup for testing purposes, but the daily MONTHLY report email never reported anything that was done on the site previously.
I still get a blank email every day, regardless of the extensions updated on the site last month.

I would have looked into it as soon as possible, but at the moment I don't even have time to breathe...

However, yes: there is something wrong, or I have set something strange (but what? just the daily cron?) or something is still not working as it should.
:(

**Reply by nikosdion** [reply]
Probably the daily CRON. Can you try setting up a daily CRON with a _daily_ report?

**Reply by morphinestyle** [reply]
Of course, I will remove the current schedule and create a daily report with a daily cron and let you know the result.

**Reply by morphinestyle** [reply]
Something has changed Nicholas.

I set up a DAILY report with a cronjob at 00:15.
I updated some extensions, and yesterday I received the email, this is the result.

The message contains numerous identical entries, so much so that Gmail truncates it for me by displaying only "CMS update found" entries.

![image](https://github.com/akeeba/panopticon/assets/35969533/6a9bf2d4-cd7a-4b7c-9497-a2876f9f2bcb)

But, if I click on "View entire message", among about 500 entries, I can finally see my 3 extension updates entries: Akeeba Backup Pro, hCaptcha and Italian Language.

![image](https://github.com/akeeba/panopticon/assets/35969533/596c458b-fee9-4afc-aee4-943b02074d3f)

**Reply by nikosdion** [reply]
If what you received by email matches what you see in the Overview, Site Reports page then it's correct.

**Reply by morphinestyle** [reply]
It doesn't make much sense to receive a very long email with only identical entries.

Among other things: the very items of interest, those of the updates, are only visible by clicking on the button at the bottom of the very long page.

It's useless like this, isn't it?

**Reply by nikosdion** [reply]
The email is correct when it contains the records you see in Overview, Site Reports for the specific site, for the previous day. If the contents of the email match the contents of that page then it's the correct email. That's what the email is supposed to do.

Whether the contents of that page _make sense_ for your use case depends on what you have configured Panopticon to do. 

If you have set up a CRON job which checks for new Joomla! versions every 3 minutes across all sites with the --force parameter then, obviously, that's a pretty nonsensical setup which creates a new log entry for the new version which is found again, and again, and again every three minutes. Does that make sense? Hell no! But that's what _you_ asked the software to do.

So, again, my software's responsibility begins and ends at doing what you **TOLD** it to do, not what you may or may not have _MEANT_ to ask it to do instead.

**Reply by morphinestyle** [reply]
I have absolutely no memory of having set up a CRON that checks for new versions of Joomla every 3 minutes.
Indeed: where can I change this parameter?

**Reply by morphinestyle** [reply]
This is the only Panopticon cron:
*/1 * * * * /usr/local/lsws/lsphp82/bin/php /home/MYWEBSITE/public_html/cli/panopticon.php task:run --loop >/dev/null 2>&1

And in the Panopticon settings I can't find any 3 minute parameter that checks the updating of Joomla sites.
Where is this 3 minute parameter located in the Panopticon settings?
